controller/network/network.go

```diff
@@ -357,18 +359,28 @@ func (network *Network) CreateSession(srcR *Router, clientId *identity.TokenId,
 	}
 }
 
-func (network *Network) selectPath(srcR *Router, svc *Service) (xt.Strategy, xt.Terminator, []*Router, error) {
-	if len(svc.Terminators) == 0 {
-		return nil, nil, nil, errors.Errorf("service %v has no Terminators", svc.Id)
+func parseServiceAndIdentity(service string) (string, string) {
```

Comment:
Minor nitpick... but the function is named "service and identity" and returns "identity and service".

Reply:
Ah, good point. I'll fix that